CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.12)
 
-project(s2-geometry)
+project(s2-geometry
+	VERSION 0.10.0)
```

Comment:
What are we allowed to put here? Can this be 0.10.0.dev or 0.10.0.alpha?

Also note that we don't actually guarantee any API stability currently, but having some version number is better than none.

Reply:
It accepts a "tweak" version, so e.g. `0.10.0.1`, but it has to be a number, it can't  be any other type of character.

Documentation: https://cmake.org/cmake/help/latest/command/project.html#command:project

Reply:
Is it good practice to put version numbers for the next release in a development version? It seems like it could lead to grief, since the API could change before the release.

Maybe put a comment to that effect.

Reply:
Ok, let's see if anyone complains this way.